ios/MembraneWebRTC.swift

```diff
@@ -490,20 +494,43 @@ class MembraneWebRTC: MembraneRTCDelegate {
         MembraneRoom.sharedInstance.endpoints.values.sorted(by: { $0.order < $1.order }).map {
             (p) -> Dictionary in
             let videoTracks = p.videoTracks.keys.map { trackId in
-                [
+                var data = [
                     "id": trackId,
                     "type": "Video",
-                    "metadata": p.tracksMetadata[trackId]?.toDict() ?? [:],
+                    "metadata": p.tracks[trackId]?.metadata.toDict() ?? [:],
                     "encoding": tracksContexts[trackId]?.encoding?.description as Any,
                     "encodingReason": tracksContexts[trackId]?.encodingReason?.rawValue as Any,
                 ]
+                
+                var simulcastConfig : SimulcastConfig? = nil
+                
+                if trackId == localAudioTrack?.trackId() ||
+                    trackId == localVideoTrack?.trackId() ||
+                    trackId == localScreencastTrack?.trackId(){
```

Comment:
you're checking if the given track is local, is there a better way to do this than 3 lines long if?